From 43f2b8dc827de98298c5233f0997d86c647694dd Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Thu, 19 Sep 2024 14:51:00 +0200
Subject: [PATCH] drm/panel: otm8009: Add flag MIPI_DSI_MODE_NO_EOT_PACKET

This flag is now managed by DSI and should be set.

Change-Id: I5e1c0f84816fab255415f93e4099e12f02c88b89
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/405686
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/gpu/drm/panel/panel-orisetech-otm8009a.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/gpu/drm/panel/panel-orisetech-otm8009a.c
+++ b/drivers/gpu/drm/panel/panel-orisetech-otm8009a.c
@@ -467,7 +467,8 @@ static int otm8009a_probe(struct mipi_ds
 	dsi->lanes = 2;
 	dsi->format = MIPI_DSI_FMT_RGB888;
 	dsi->mode_flags = MIPI_DSI_MODE_VIDEO | MIPI_DSI_MODE_VIDEO_BURST |
-			  MIPI_DSI_MODE_LPM | MIPI_DSI_CLOCK_NON_CONTINUOUS;
+			  MIPI_DSI_MODE_LPM | MIPI_DSI_CLOCK_NON_CONTINUOUS |
+			  MIPI_DSI_MODE_NO_EOT_PACKET;
 
 	pm_runtime_enable(ctx->dev);
 	pm_runtime_set_autosuspend_delay(ctx->dev, 1000);
